pkg/csi/service/node.go

```diff
@@ -648,25 +648,25 @@ func (driver *vsphereCSIDriver) NodeGetInfo(
 
 	nodeID := os.Getenv("NODE_NAME")
 	if nodeID == "" {
-		return nil, logger.LogNewErrorCode(log, codes.Internal, "ENV NODE_NAME is not set")
+		return nil, logger.LogNewErrorCode(log, codes.Internal, "env NODE_NAME is not set")
 	}
 	var maxVolumesPerNode int64
 	if v := os.Getenv("MAX_VOLUMES_PER_NODE"); v != "" {
 		if value, err := strconv.ParseInt(v, 10, 64); err == nil {
 			if value < 0 {
 				return nil, logger.LogNewErrorCodef(log, codes.Internal,
-					"NodeGetInfo: MAX_VOLUMES_PER_NODE set in env variable %v is less than 0", v)
+					"nodeGetInfo: MAX_VOLUMES_PER_NODE set in env variable %v is less than 0", v)
```

Comment:
Should a noun start with lower case? NodeGetInfo is a noun, so I'm not sure if we need to start everything with a lower case. But I'm ok with this as I dont know what is the guideline in such cases.

Reply:
I found something on my question - https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

Reply:
Sandeep, can you file a JIRA to cleansing the error messages? It deals with business logic, but better handled by the code owner.

Reply:
Yes, done.

Reply:
Based on the guideline, I think "NodeGetInfo" or "ENV" don't need to start with a lower case while "volume" should start with a lower case.

Reply:
Done, @xing-yang . I will leave the rest to the code owner.